packages/block-editor/src/hooks/layout.js

```diff
@@ -63,10 +63,13 @@ export function useLayoutClasses( block = {} ) {
 		globalLayoutSettings?.definitions?.[ usedLayout?.type || 'default' ]
 			?.className
 	) {
-		layoutClassnames.push(
+		const baseClassName =
 			globalLayoutSettings?.definitions?.[ usedLayout?.type || 'default' ]
-				?.className
-		);
+				?.className;
+		const compoundClassName = `wp-block-${ name
+			.split( '/' )
```

Comment:
Similar to the comment on the PHP code, can we use `getBlockDefaultClassName( name )` here to get the classname?